kfs-core/src/main/java/edu/arizona/kfs/sys/businessobject/options/PaymentMethodValuesFinder.java

```diff
@@ -0,0 +1,56 @@
+package edu.arizona.kfs.sys.businessobject.options;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.util.KeyValue;
+import org.kuali.rice.core.api.util.ConcreteKeyValue;
+import org.kuali.rice.krad.keyvalues.KeyValuesBase;
+import org.kuali.rice.krad.service.BusinessObjectService;
+
+import edu.arizona.kfs.fp.businessobject.PaymentMethod;
+
+/**
+ * This class returns list of payment method key value pairs.
+ * 
+ * Customization for UA: addition of "A" type for credit card payments.
+ * 
+ * @author jonathan
+ * @see org.kuali.kfs.fp.businessobject.options.PaymentMethodValuesFinder
+ */
+public class PaymentMethodValuesFinder extends KeyValuesBase {
+    private static BusinessObjectService businessObjectService;
+    static protected Map<String,String> filterCriteria = new HashMap<String, String>();
+    static {
+        filterCriteria.put("active", "Y");
+    }
+    
+    /*
+     * @see org.kuali.keyvalues.KeyValuesFinder#getKeyValues()
```

Comment:
Please update comment as class was moved in KFS 6.0:
see org.kuali.rice.krad.keyvalues.KeyValuesFinder#getKeyValues()